core-library/src/main/java/org/silverpeas/core/admin/persistence/InstanceDataTable.java

```diff
@@ -38,12 +38,25 @@
  */
 public class InstanceDataTable extends Table<InstanceDataRow> {
 
+  private static final String INSTANCEDATA_COLUMNS = "id,componentId,name,label,value";
+  private static final String INSERT_INSTANCEDATA = "insert into ST_Instance_Data("
+      + INSTANCEDATA_COLUMNS + ") values (?,?,?,?,?)";
+  private static final String SELECT_ALL_COMPONENTS_BY_PARAMETER_VALUE =
+      "select " + INSTANCEDATA_COLUMNS +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "select " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)